src/fiatconnect/consts.ts

```diff
@@ -0,0 +1,42 @@
+import { CryptoType, FiatType } from '@fiatconnect/fiatconnect-types'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { CiCoCurrency } from 'src/utils/currencies'
+
+export const FIATCONNECT_CURRENCY_TO_WALLET_CURRENCY = {
+  [FiatType.ARS]: LocalCurrencyCode.ARS,
+  [FiatType.BOB]: LocalCurrencyCode.BOB,
+  [FiatType.BRL]: LocalCurrencyCode.BRL,
+  [FiatType.CLP]: LocalCurrencyCode.CLP,
+  [FiatType.CNY]: LocalCurrencyCode.CNY,
+  [FiatType.COP]: LocalCurrencyCode.COP,
+  [FiatType.EUR]: LocalCurrencyCode.EUR,
+  [FiatType.FKP]: LocalCurrencyCode.FKP,
+  [FiatType.GBP]: LocalCurrencyCode.GBP,
+  [FiatType.GHS]: LocalCurrencyCode.GHS,
+  [FiatType.GNF]: LocalCurrencyCode.GNF,
+  [FiatType.GYD]: LocalCurrencyCode.GYD,
+  [FiatType.INR]: LocalCurrencyCode.INR,
+  [FiatType.KES]: LocalCurrencyCode.KES,
+  [FiatType.MXN]: LocalCurrencyCode.MXN,
+  [FiatType.NGN]: LocalCurrencyCode.NGN,
+  [FiatType.PAB]: LocalCurrencyCode.PAB,
+  [FiatType.PEN]: LocalCurrencyCode.PEN,
+  [FiatType.PHP]: LocalCurrencyCode.PHP,
+  [FiatType.PYG]: LocalCurrencyCode.PYG,
+  [FiatType.RWF]: LocalCurrencyCode.RWF,
+  [FiatType.SRD]: LocalCurrencyCode.SRD,
+  [FiatType.UGX]: LocalCurrencyCode.UGX,
+  [FiatType.USD]: LocalCurrencyCode.USD,
+  [FiatType.UYU]: LocalCurrencyCode.UYU,
+  [FiatType.VES]: LocalCurrencyCode.VES,
+  [FiatType.XAF]: LocalCurrencyCode.XAF,
+  [FiatType.XOF]: LocalCurrencyCode.XOF,
+  [FiatType.ZAR]: LocalCurrencyCode.ZAR,
+}
+
+export const WALLET_CRYPTO_TO_FIATCONNECT_CRYPTO = {
```

Comment:
```suggestion
export const WALLET_CRYPTO_TO_FIATCONNECT_CRYPTO: Record<CiCoCurrency, CryptoType> = {
```
an explicit type is important to add here because it gives us compile-time guarantees that if a CiCoCurrency is added, the set of keys in this object must be exhaustive, else an error will be thrown

Reply:
Makes sense, good catch